### PR TITLE
portability: patch mac os dns issues

### PR DESCRIPTION
**Pull request by oliver-sanders**
sibling: https://github.com/cylc/cylc-flow/pull/4296

### CODE REVIEW

File: src/installation.rst
Comment (kinow) [inline src/installation.rst:104]: :+1: :heavy_check_mark:  done!

**Review by kinow — APPROVED** [review]
Looks good. The CI error is not related I think

```
2021-07-09T09:14:07Z ERROR - Workflow shutting down - AUTOMATIC(ON-TASK-FAILURE)
2021-07-09T09:14:07Z WARNING - Partially satisfied prerequisites left over:
	consolidate_observations.20210709T0800Z is waiting on:
	* get_observations_heathrow.20210709T0800Z succeeded
2021-07-09T09:14:07Z WARNING - Orphaned task jobs:
	* get_rainfall.20210709T0800Z (running)
2021-07-09T09:14:08Z INFO - DONE
Error: Process completed with exit code 1.
```

**Comment by kinow** [comment]
(not sure whether we can merge it now, or if we need to wait for the sibling PR; if it's OK, feel free to press merge pull request or just confirm and I'll do it later :+1: thanks!)

**Comment by oliver-sanders** [comment]
Should wait until the other PR is merged.

**Comment by oliver-sanders** [comment]
sibling PR merged, rebased, test passing (ignore tut workflow for now), merging.

**Pull request merged**
